packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -1,13 +1,57 @@
 /* eslint-disable class-methods-use-this */
+const expect = require('expect');
 const axe = require('../commands/run-axe');
+const viewportHelpers = require('../commands/viewport-helpers');
+const hideInputCaret = require('../commands/hide-input-caret');
 
 class TerraService {
   /**
    * Service hook executed prior to test execution.
    * Initializes the Terra Service's custom commands.
    */
-  before() {
+  before(capabilities) {
     global.browser.addCommand('axe', axe);
+
+    /* Add the Jest expect module the use the Jest matchers. */
+    global.expect = expect;
```

Comment:
Isn't this provided by default? Why do we need to add this explicitly?

Reply:
This is not available in the before hook but we need it below to setup the Terra commands.